### ISSUE TITLE
chapter 11 reusing only the lower layers 

### ISSUE THREAD

**Issue by nitml** [ageron/handson-ml#273]
our original model have was having 5 hidden layer and we wanted to reuse only lower 3 hidden layer and build a hidden layer 4 on top of it so code in jupyter notebook was : 
`  reset_graph()

 n_hidden4 = 20  # new layer
 n_outputs = 10  # new layer

 saver = tf.train.import_meta_graph("./my_model_final.ckpt.meta")

 X = tf.get_default_graph().get_tensor_by_name("X:0")
 y = tf.get_default_graph().get_tensor_by_name("y:0")

 hidden3 = tf.get_default_graph().get_tensor_by_name("dnn/hidden4/Relu:0")

new_hidden4 = tf.layers.dense(hidden3, n_hidden4, activation=tf.nn.relu, name="new_hidden4")
new_logits = tf.layers.dense(new_hidden4, n_outputs, name="new_outputs")`

and original model graph in "dnn "was : 
![capture](https://user-images.githubusercontent.com/40690661/43645111-95dab168-974e-11e8-83ba-0633c314fa56.JPG)

My question is why we have used "dnn/hidden4/Relu:0" in place of "dnn/hidden3/Relu:0" as this will give output of activation function of hidden3 ?

**Comment by ageron** [comment]
Good catch! Indeed, this is an error. If we want to reuse layers up to layer 3, we should get the output of layer 3, not 4.  I'll fix this, thanks!
